src/map.cpp

```diff
@@ -551,18 +565,33 @@ bool Map::checkSightLine(uint16_t x0, uint16_t y0, uint16_t x1, uint16_t y1, uin
 		return true;
 	}
 
+	// from top to bottom
+	float modifier = 0;
 	if (std::abs(y1 - y0) > std::abs(x1 - x0)) {
 		if (y1 > y0) {
-			return checkSteepLine(y0, x0, y1, x1, z);
+			if (!g_config.getBoolean(ConfigManager::CLASSIC_ATTACK_SPEED) && x0 != x1) {
```

Comment:
What does CLASSIC_ATTACK_SPEED have to do with this?

Reply:
Initial confounding. This was meant to be `ORIGINAL_SIGHT_CALCULATION`, which will be removed regardless in favor of precompiler code substitution.